### ISSUE TITLE
estimate_contrasts() : Report CI adjustment in footnote (and reconsider default)

### ISSUE THREAD

**Issue by bwiernik** [easystats/modelbased#190]
Currently, `estimate_contrasts()` applies a CI adjustment (straight bonferonni) but does not report it in the results footnote. If the CI is adjusted, we should report that in addition to the p-value adjustment.

```r
d <- transform(mtcars, cyl = factor(cyl))
m <- lm(mpg ~ cyl, data = d)

# CIs are adjusted, but this is not reported in the footnote
modelbased::estimate_contrasts(m)

# No CI adjustment
modelbased::estimate_contrasts(m, adjust = "none")
```

Do we want to apply an adjustment at all by default? We only do this for `estimate_contrasts()` and `correlation::correlation()`, not other functions in the easystats 'verse. That seems a little inconsistent? If we do want an adjustment by default, for `estimate_contrasts()`, perhaps `dunnettx` or `mvt` (or `mvt` when the number of row is small and `dunnettx` otherwise) would be better by default than `holm` because they are less conservative?

See also other issues with `adjust`: https://github.com/easystats/modelbased/issues/163

**Comment by DominiqueMakowski** [comment]
@bwiernik I don't have strong opinions about the defaults. But indeed, more and more I don't like applying stuff that the user didn't explicitly asked, so I could be quite easily convinced to have `p_adjust="none"` by default wdyt?

**Comment by DominiqueMakowski** [comment]
@strengejacke ?

**Comment by strengejacke** [comment]
Agree. That's in line with _marginaleffects_, and if we decide on a value, we have to do it like emmeans and find the most appropriate adjustement-method based on the kind of comparison and number of tests. It's better to leave this to the user.

**Comment by DominiqueMakowski** [comment]
alright
